python/tvm/driver/tvmc/autotuner.py

```diff
@@ -286,17 +290,18 @@ def drive_tune(args):
         )
 
     if args.enable_autoscheduler:
-        # Specify hardware parameters
-        hardware_params = auto_scheduler.HardwareParams(
-            args.num_cores,
-            args.vector_unit_bytes,
-            args.cache_line_bytes,
-            args.max_shared_memory_per_block,
-            args.max_local_memory_per_block,
-            args.max_threads_per_block,
-            args.max_vthread_extent,
-            args.warp_size,
-        )
+        if args.enable_hardware_params:
+            # Specify hardware parameters
+            hardware_params = auto_scheduler.HardwareParams(
```

Comment:
What is the value for `hardware_params`, used in line 310 below, in case `args.enable_hardware_params` is `False`?

Reply:
When not specified, the default hardware params should be None. I mistakenly didn't include this earlier.